queueing.h

```diff
@@ -0,0 +1,203 @@
+/* SPDX-License-Identifier: GPL-2.0 */

```

Comment:
Instead of adding this file in repo, I think it's better to download it from the WG repo during installation, e.g. fetch it from https://git.zx2c4.com/wireguard-linux/plain/drivers/net/wireguard/queueing.h

Reply:
Mike, thanks for the suggestion. Indeed it is better to download the file.